src/Database.hs

```diff
@@ -120,15 +120,15 @@ initLastBlock ::
   ) =>
   BlockInfo ->
   m ()
-initLastBlock (BlockInfo slot hash) = do
```

Comment:
Functions that save block in db, shouldn't accept `BlockInfo` now but explicit slot and hash, it will break after reading invalid slot and hashes from db after trying to insert `origin`